sympy/integrals/manualintegrate.py

```diff
@@ -1528,6 +1535,23 @@ def eval_reciprocal_sqrt_quadratic(a, b, c, integrand, x):
     return log(2*sqrt(c)*sqrt(a+b*x+c*x**2)+b+2*c*x)/sqrt(c)
 
 
+@evaluates(SqrtQuadraticDenomRule)
+def eval_sqrt_quadratic_denom(a, b, c, coeffs: list[Expr], integrand, x):
+    result_coeffs = []
```

Comment:
It would be nice to have some comments or docstring somewhere to explain what this rule is or what the parameters mean. This file seems to be completely absent of any explanation but that should be something to improve.

Reply:
Using tons of latex seems the only readable way for that, but latex doesn't fit inline comment.
I guess this wasn't addressed here because readable steps were provided anyway at SymPy Gamma, so no need to be documented here.
The PR description is always here, and I'll implement readable steps for it at SymPy Beta after a new SymPy release.

Reply:
It would just be nice to have something that says what kind of integrand this is used for and how the coefficients relate to that integrand.

Reply:
I would just write a docstring that uses dollar math with latex. We can include it in the developer documentation to make it easier to read.